README.md

```diff
@@ -5,6 +5,16 @@ Leetcode exercise for training. A solution with multiple console applications.
 
 Run `dotnet build` to build the project.
 
+## Test
```

Comment:
A good practice in the README is to mention the minimum version you need to run the project. For example:

## Build

To build this project you need at least NetCore 3.1.
Then you can build the project with

```bash
dotnet build
```

Reply:
Perfect! :)